### PR TITLE
new TimerDialog package

### PR DESCRIPTION
**Pull request by Overkane**
Wrappers for Timer Dialog API from common.j and small fixes in Angle package

### CODE REVIEW

File: wurst/_handles/TimerDialog.wurst [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+public function timer.createTimerDialog() returns timerdialog
+    return CreateTimerDialog(this)
+
+public function timerdialog.destr() 
+    DestroyTimerDialog(this)
+
+public function timerdialog.setTitle(string title) 
+    TimerDialogSetTitle(this, title)
+
+public function timerdialog.setTitleColor(int red, int green, int blue, int alpha)
```
Comment (Frotty) [inline wurst/_handles/TimerDialog.wurst]: please add an overloaded version of this function taking a colorA tuple.

Comment (Overkane) [reply wurst/_handles/TimerDialog.wurst]: Done

File: wurst/_handles/TimerDialog.wurst [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+public function timerdialog.setTitleColor(colorA colorA) 
+    TimerDialogSetTitleColor(this, colorA.red, colorA.green, colorA.blue, colorA.alpha)
+
+public function timerdialog.setTimeColor(int red, int green, int blue, int alpha) 
+    TimerDialogSetTimeColor(this, red, green, blue, alpha)
+
+public function timerdialog.setTimeColor(colorA colorA) 
+    TimerDialogSetTimeColor(this, colorA.red, colorA.green, colorA.blue, colorA.alpha)
+
+public function timerdialog.setSpeed(real speedMultFactor)
```
Comment (Cokemonkey11) [inline wurst/_handles/TimerDialog.wurst]: Can you add a doc comment for what this is for? I'm not familiar with the native

File: wurst/math/Angle.wurst [hunk 3]
```diff
@@ -42,8 +42,8 @@ public function int.asAngleRadians() returns angle
 /** interpret this real as an angle given in radians*/
 public function real.asAngleRadians() returns angle
 	return angle(this)
-/** get the sinus of this angle */
 
+/** get the sinus of this angle */
```
Comment (Cokemonkey11) [inline wurst/math/Angle.wurst]: /sinus/sine/

**Comment by Cokemonkey11** [comment]
LGTM

**Comment by Frotty** [comment]
Nice 🍬

**Pull request merged**